templates/cassiopeia/scss/vendor/choicesjs/choices.scss

```diff
@@ -8,125 +8,170 @@
     left: 0;
-    display: block;
+    display: flex;
+    justify-content: center;
+    align-items: center;
     text-align: center;
+    line-height: 1.5;
     text-indent: 0;
     content: "\00d7";
   }
 
   &:focus {
     outline: none;
   }
+
+  &:hover,
+  &:focus {
+    opacity: 1;
+  }
 }
 
-.choices[data-type*="select-one"] {
-  .choices__button_joomla {
-    position: absolute;
-    top: 50%;
-    right: 0;
-    width: 20px;
-    height: 20px;
-    padding: 0;
-    margin-top: -10px;
-    margin-right: 45px;
-    border-radius: 10em;
-    opacity: .5;
-
-    [dir=rtl] & {
-      right: auto;
-      left: 0;
-      margin-right: 0;
-      margin-left: 45px;
+.choices {
+  font-size: 1rem;
+  line-height: 1.5;
+
+  .choices__inner {
+    min-height: calc(1.5em + 1.2rem + 2px);
+  }
+
+  &[data-type*="select-one"] {
+    .choices__inner {
+      padding: $input-btn-padding-y $custom-select-indicator-padding $input-btn-padding-y $input-btn-padding-x;
+      background: $custom-select-bg url("../../../images/select-bg.svg") no-repeat right center / $custom-select-bg-size;
+
+      [dir=rtl] & {
+        padding: $input-btn-padding-y $input-btn-padding-x $input-btn-padding-y $custom-select-indicator-padding;
+        background: $custom-select-bg url("../../../images/select-bg-rtl.svg") no-repeat left center / $custom-select-bg-size;
+      }
     }
 
-    &:hover,
-    &:focus {
-      opacity: 1;
+    .choices__button_joomla {
+      position: absolute;
+      right: 0;
+      width: 1.25em;
+      height: 1.25em;
+      padding: 0;
+      border-radius: 10em;
+      opacity: .5;
+
+      [dir=rtl] & {
+        right: auto;
+        left: 0;
+      }
+
+      &:focus {
+        box-shadow: 0 0 0 2px hsl(187, 100%, 42%);
+      }
     }
 
-    &:focus {
-      box-shadow: 0 0 0 2px hsl(187, 100%, 42%);
+    &::after {
+      display: none;
     }
   }
 
-  &::after {
-    display: none;
-  }
+  &[data-type*="select-multiple"],
+  &[data-type*="text"] {
+    .choices__inner {
+      padding: .375 * $input-btn-padding-y .25 * $input-btn-padding-x .125 * $input-btn-padding-y;
 
-  .choices__inner {
-    padding: $custom-select-padding-y ($custom-select-padding-x + $custom-select-indicator-padding) $custom-select-padding-y $custom-select-padding-x;
-    background: url("../../../images/select-bg.svg") no-repeat right center / $custom-select-bg-size;
-    background-color: $custom-select-bg;
-
-    [dir=rtl] & {
-      padding: $custom-select-padding-y $custom-select-padding-x $custom-select-padding-y ($custom-select-padding-x + $custom-select-indicator-padding);
-      background: url("../../../images/select-bg-rtl.svg") no-repeat left center / $custom-select-bg-size;
-      background-color: $custom-select-bg;
+      [dir=rtl] & {
+        text-align: right;
+      }
+    }
+
+    .choices__button_joomla {
+      position: relative;
+      display: inline-block;
+      width: .5em;
+      padding-left: 1em;
+      margin: 0 -1 * .25em 0 .5em;
```

Comment:
I thought margin only accepted 4 values?

Reply:
perhaps that's the issue... wrong calculation
```
margin-left: 0;
margin-right: -1 * .25em;
margin-bottom: 0;
margin-left: .5em;
```